addons/account/static/src/components/section_and_note_fields_backend/section_and_note_fields_backend.js

```diff
@@ -42,32 +43,40 @@ export class SectionAndNoteListRenderer extends ListRenderer {
 
     getCellClass(column, record) {
         const classNames = super.getCellClass(column, record);
-        if (this.isSectionOrNote(record) && column.widget !== "handle" && column.name !== "name") {
+        if (this.isSectionOrNote(record) && column.widget !== "handle" && column.name !== this.titleField) {
             return `${classNames} o_hidden`;
         }
         return classNames;
     }
 
-    getCellColspan(column) {
-        if (!this.isSectionOrNote() || column.widget === "handle") {
-            return 1;
+    getColumns(record) {
+        const columns = super.getColumns(record);
+        if (this.isSectionOrNote(record)) {
+            return this.getSectionColumns(columns);
         }
-        let nbrColumns = this.withHandleColumn ? this.state.columns.length + 1 : this.state.columns.length;
-        let withTrashColumn = this.props.activeActions && (this.props.activeActions.canDelete || this.props.activeActions.canUnlink || 0);
-        return nbrColumns - this.withHandleColumn - withTrashColumn;
+        return columns;
     }
-}
-SectionAndNoteListRenderer.template = "account.sectionAndNoteListRenderer";
-SectionAndNoteListRenderer.recordRowTemplate = "account.SectionAndNoteListRenderer.RecordRow";
 
-export class SectionAndNoteFieldOne2Many extends X2ManyField {
-    setup() {
-        super.setup();
-        if (this.viewMode === 'list') {
-            this.Renderer = SectionAndNoteListRenderer;
+    getSectionColumns(columns) {
+        const sectionColumns = columns.filter((col) => col.widget === "handle");
+        let colspan = columns.length - sectionColumns.length;
+        if (this.props.activeActions.onDelete) {
+            colspan++;
```

Comment:
Aren't you supposed to remove a span? 
Don't we want to have the possibility to remove the line?

Reply:
With a delete button / or without, the colspan should be the same